<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5CurrencyCrossRateRecord</class>
 <widget class="QWidget" name="CE5CurrencyCrossRateRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="leCurr_2">
     <property name="Field" stdset="0">
      <string>f_currency2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="C5LineEditWithSelector" name="leCurrName_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_currencyname2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5DateEdit" name="leDate">
     <property name="Field" stdset="0">
      <string>f_date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="C5LineEditWithSelector" name="leCurrName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_currencyname1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Currency 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Currency 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leCurr">
     <property name="Field" stdset="0">
      <string>f_currency1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Rate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit">
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="Primary" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="C5LineEditWithSelector" name="leRate">
     <property name="Field" stdset="0">
      <string>f_rate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chSetCurrentRate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Set current rate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>leDate</tabstop>
  <tabstop>leCurr</tabstop>
  <tabstop>leCurr_2</tabstop>
  <tabstop>leRate</tabstop>
  <tabstop>chSetCurrentRate</tabstop>
  <tabstop>leCurrName_2</tabstop>
  <tabstop>leCurrName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
